gnat2goto/driver/arrays.ads

```diff
@@ -81,6 +92,12 @@ private
    function Make_Array_Index_Op
      (Base_Irep : Irep; Base_Type : Node_Id; Idx_Irep : Irep) return Irep;
 
+   function Build_Array_Size (First : Irep; Last : Irep; Idx_Type : Irep)
+                              return Irep
+     with Pre => (Kind (First) in Class_Expr
+                  and then Kind (Last) in Class_Expr
```

Comment:
FWIW my preference is to use `and` instead of `and then` if there's no order dependence of the expressions

Reply:
Changed.